.drone.yml

```diff
@@ -3,10 +3,49 @@ workspace:
   path: src/github.com/drone-plugins/drone-s3-cache
 
 pipeline:
-  build:
-    image: golang:1.7
-    environment:
-      - CGO_ENABLED=0
+  test:
+    image: golang:1.9
+    group: testing
     commands:
+      - go vet
       - go test -cover -coverprofile=coverage.out
-      - go build -ldflags "-s -w -X main.revision=$(git rev-parse HEAD)" -a
+
+  build:
+    image: golang:1.9
+    group: testing
+    commands: sh .drone.sh
+
+  publish_linux_amd64:
+    image: plugins/docker
+    group: tags
+    repo: plugins/s3-cache
+    tags: [ "latest", "1.1", "1" ]
+    secrets: [ docker_username, docker_password ]
+    when:
+      branch: master
+      event: push
+      status: success
```

Comment:
I don't think that you need that, this should be the default

Reply:
Done. @tboerger